src/style.css

```
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Font1";
+  src: url("./fonts/Avenir.ttc");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Font2";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 300;
+}
+
+form {
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+label {
+  display: flex;
+  height: 100%;
+}
+
+input {
+  width: 100%;
+  border: none;
+  outline: none;
+  font-family: "Font1", sans-serif;
```

```suggestion
  font-family: inherit;
```
set it to html and inherit here